lms/envs/aws.py

```diff
@@ -176,7 +176,11 @@
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
```

Comment:
To better explain my position here - what I really want here is to avoid catching Open edX users by surprise. I think defaulting the production behavior to sometimes [present](https://github.com/edx/edx-platform/blob/a9c548c3fa814b8f12f88b1ffbbd14d3189a12a9/lms/djangoapps/bulk_email/tasks.py#L105) [URLs](https://github.com/edx/edx-platform/blob/bd87061f1cadab0c1c22a86cc5b76094a3742f27/common/djangoapps/util/course.py#L23) to `http://localhost:8000/` instead of just erroring out and saying that needs to be done immediately will catch some operators by surprise. ;)

On another note: If the configuration repo defaults to `localhost` for `LMS_ROOT_URL`, it probably should default to `None` instead.